src/features/eighteenBirthday/ShowEligibleCardGuesser.test.tsx

```diff
@@ -0,0 +1,110 @@
+import { act, render } from '@testing-library/react-native'
+import { rest } from 'msw/'
+import React from 'react'
+import waitForExpect from 'wait-for-expect'
+
+import { navigate } from '__mocks__/@react-navigation/native'
+import { api } from 'api/api'
+import { AuthContext } from 'features/auth/AuthContext'
+import { env } from 'libs/environment'
+import { storage } from 'libs/storage'
+import { reactQueryProviderHOC } from 'tests/reactQueryProviderHOC'
+import { server } from 'tests/server'
+import { flushAllPromises } from 'tests/utils'
+
+import { ShowEligibleCardGuesser } from './ShowEligibleCardGuesser'
+
+describe('<ShowEligibleCardGuesser />', () => {
+  afterEach(async () => {
+    jest.clearAllTimers()
+    await storage.clear('has_seen_eligible_card')
+    await act(flushAllPromises)
+  })
+
+  it('should always return a null component', async () => {
+    jest.useFakeTimers()
+    const renderAPINotLoggedIn = renderShowEligibleCardGuesser({ isLoggedIn: false })
+
+    await act(flushAllPromises)
+    jest.advanceTimersByTime(1000)
+    expect(renderAPINotLoggedIn.toJSON()).toBeNull()
+
+    const renderAPI = renderShowEligibleCardGuesser()
+
+    await act(flushAllPromises)
+    jest.advanceTimersByTime(1000)
+    expect(renderAPI.toJSON()).toBeNull()
```

Comment:
Tu n'as pas besoin de faker des timers ici :
```suggestion
    const renderAPINotLoggedIn = renderShowEligibleCardGuesser({ isLoggedIn: false })

    await act(flushAllPromises)
    expect(renderAPINotLoggedIn.toJSON()).toBeNull()

    const renderAPI = renderShowEligibleCardGuesser()

    await act(flushAllPromises)
    expect(renderAPI.toJSON()).toBeNull()
```

Reply:
ok